cmd/jujud/agent/machine_test.go

```diff
@@ -895,6 +913,26 @@ func (s *MachineSuite) TestMachineAgentSymlinks(c *gc.C) {
 	s.waitStopped(c, state.JobManageModel, a, done)
 }
 
+func (s *MachineSuite) TestEnsureMachineAgentDir(c *gc.C) {
+	stm, _, _ := s.primeAgent(c, state.JobManageModel)
+	s.PatchValue(&series.MustHostSeries, func() string { return "trusty" })
```

Comment:
Strictly speaking, we need to choose a series that is different to the host on which the test is run. If the test is run on trusty, we need to choose something else.

Reply:
changed to quantal